modules/multi-runner/variables.tf

```
@@ -515,13 +515,9 @@ variable "workflow_job_queue_configuration" {
 variable "ssm_paths" {
   description = "The root path used in SSM to store configuration and secreets."
   type = object({
-    root    = string
-    app     = string
-    runners = string
+    root    = optional(string, "github-action-runners")
+    app     = optional(string, "app")
+    runners = optional(string, "runners")
   })
-  default = {
-    root    = "github-action-runners"
-    runners = "runners"
-    app     = "app"
-  }
+  default = {}
```

I am not sure what is the use of this?

---

Is not used, leaving the default let it fail.

---

With this example you can try it your self,

```ncl
variable "test" {
  type = object({
    name = optional(string, "jonh")
    age  = optional(number, 20)
  })
  default = {}
}

output "test" {
  value = var.test
}
```

By setting the default to `{}` the object get a required value, the optionals ensures if object values are not set, they set with the default.